tools/modlog/converter.ts

```diff
@@ -388,6 +388,14 @@ export function modernizeLog(line: string, nextLine?: string): string | undefine
 			const [, user] = log.split(' by ');
 			return `TOUR SCOUT: by ${toID(user)}: disallow`;
 		},
+		"[thetournament] was set to disallowing non-autoconfirmed users' joining": (log) => {
+			const [, user] = log.split(' by ');
+			return `TOUR AUTOCONFIRMEDONLY: by ${toID(user)}: on`;
+		},
+		"[thetournament] was set to allowing non-autoconfirmed users' joining": (log) => {
+			const [, user] = log.split(' by ');
+			return `TOUR AUTOCONFIRMEDONLY: by ${toID(user)}: off`;
+		},
```

Comment:
Not necessary, we don't need to modernize old entries for this since there _are_ no old entries - this is a new command.

Reply:
I did not understand what "old" means - I'm removing these changes.